src/statefulmodelinstance.hpp

```diff
@@ -30,18 +30,21 @@ const uint32_t SEQUENCE_END = 2;
 
 class StatefulModelInstance : public ModelInstance {
     static constexpr std::array<const char*, 2> SPECIAL_INPUT_NAMES{"sequence_id", "sequence_control_input"};
-    SequenceManager sequenceManager;
+    std::unique_ptr<SequenceManager> sequenceManager;
     bool performLowLatencyTransformation;
 
 public:
     /**
          * @brief A default constructor
          */
     StatefulModelInstance(const std::string& name, model_version_t version) :
-        ModelInstance(name, version) {}
+        ModelInstance(name, version) {
+        sequenceManager = std::make_unique<SequenceManager>(config.getSequenceTimeout(), config.getMaxSequenceNumber());
+    }
 
     SequenceManager& getSequenceManager() {
```

Comment:
Can we return reference to the pointer that we store as StatefulModelInstance member?